dowhy/timeseries/causal_effect_tutorial.ipynb

```diff
@@ -99,27 +79,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "([5, 7], [1, 3])"
-      ]
-     },
-     "execution_count": 4,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "# read the dataframe in a csv format from the user\n",
     "dataset_path=\"../../docs/source/example_notebooks/datasets/temporal_dataset.csv\"\n",
     "dataframe=pd.read_csv(dataset_path)\n",
     "\n",
     "# the node for which effect estimation has to be done, node:6\n",
-    "target_node = 6\n",
+    "target_node = 'V6'\n",
     "\n",
     "# find the action nodes of the given target node with respective lag times\n",
     "parents = find_lagged_parent_nodes(graph, target_node)\n",
```

Comment:
for readability, instead of `parents`, expand it to `parents`, `time_lags` so that it is intuitive for reader